packages/components/radio-group/src/radio-group.ts

```diff
@@ -185,6 +186,12 @@ export class RadioGroup<T = unknown> extends FormControlMixin(LitElement) {
     this.#rovingTabindexController.focus();
   }
 
+  #onClick(event: Event): void {
+    if (event.target === this) {
```

Comment:
sometimes radio group is getting focus, sometimes not when clicking on the label
https://kind-plant-088739303-1536.westeurope.1.azurestaticapps.net/?path=/story/form-form--all

Reply:
same with `select`

Reply:
Discussed on slack: this is the browser determining if `:focus-visible` applies.